flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1067,6 +1062,9 @@ class FMCMainDisplay extends BaseAirliners {
             }
             SimVar.SetSimVarValue("L:A320_FCU_SHOW_SELECTED_HEADING", "number", 1);
         }
+        if (_event === "VS") {
+            this.flightPhaseManager.handleFcuVSKnob();
```

Comment:
Ok, now I get why the timeout is there. But is the event ever needed? Why not just "listening" to the modes and when the respective mode is engaged it switches to DESCEND phase.

Reply:
The entire system is event based, there are some exceptions, e.g. where timers are needed.